macbook.md

```diff
@@ -111,34 +111,14 @@ sudo apt-get install bcmwl-kernel-source
 
 If you don't have a tethered network connection, download these two files from some other computer:
 
-* [`dkms` for Ubuntu 15.10](http://packages.ubuntu.com/wily/all/dkms/download)
-* [`bcmwl-kernel-source` for Ubuntu 15.10](http://packages.ubuntu.com/wily/amd64/bcmwl-kernel-source/download)
+* [`dkms` for Ubuntu 15.10](http://packages.ubuntu.com/xenial/all/dkms/download)
+* [`bcmwl-kernel-source` for Ubuntu 15.10](http://packages.ubuntu.com/xenial/amd64/bcmwl-kernel-source/download)
 
 Transfer the two `.deb` files via USB to your Macbook, then install them with `dpkg`:
 
 ```
 sudo dpkg -i dkms_2.2.0.3-2ubuntu6_all.deb
-sudo dpkg -i bcmwl-kernel-source_6.30.223.248+bdcom-0ubuntu7_amd64.deb
-```
-
-#### GRUB maintenance
-
-This GRUB change fixes a reported occasional SSD freeze bug. (I've never seen it, but I'm following orders out of an abundance of caution.)
-
-```bash
-sudo nano /etc/default/grub
-```
-
-Change the `GRUB_CMDLINE_LINUX=""` line to read:
-
-```
-GRUB_CMDLINE_LINUX="libata.force=noncq"
-```
-
-Finally, save and update GRUB to make those two changes take effect:
-
-```bash
-sudo update-grub
```

Comment:
I also removed the section about updating grub to fix a SSD crash bug, I'd like to just wait and see if this ever actually happens.